.gitignore

```diff
@@ -1,2 +1,4 @@
 MultiTestDB.conf
-.idea
\ No newline at end of file
+.idea
+/_Deparsed_XSubs.pm
+E
```

Comment:
I don't object to additions to .gitignore, but this looks a bit like a mistake?

Reply:
for the E yes.